app/domain/operations/products/import_service_area.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+module Operations
+  module Products
+    # This class is to load service areas.
+    class ImportServiceArea
+      include Dry::Monads[:result, :do]
+
+      def call(params)
+        values               = yield validate(params)
+        issuer_profiles_hash = yield load_issuer_profiles
+        sheet                = yield load_data(values[:file], values[:rating_area_model])
+        _load                = yield import_records(sheet, values[:year], values[:row_data_begin], issuer_profiles_hash, values[:rating_area_model])
+        Success()
+      end
+
+      private
+
+      def validate(params)
+        return Failure('Missing File') if params[:file].blank?
+        return Failure('Missing Year') if params[:year].blank?
+        return Failure('Missing Row data begins with') if params[:row_data_begin].blank?
+        params.merge!(rating_area_model: EnrollRegistry[:enroll_app].setting(:geographic_rating_area_model).item)
+        Success(params)
```

Comment:
Here we should check if the data they gave us actually corresponds to a 'real' county, zip, our county/zip pair.

We get invalid data all the time - how will we know what row is wrong?  Why is it wrong?